src/utils/get-ts-config.js

```diff
@@ -51,6 +52,26 @@ const getTsConfig = async (memo, fs, sourcePath, projectRoot) => {
     }
 };
 
+/**
+ * @param {string} jsoncString
+ * @param {string?} fileName
+ * @returns {any}
+ */
+function parseJsoncOrThrow(jsoncString, fileName){
+    /**
+     * @type {JSONC.ParseError[]}
+    */
+    let errors = [];
+    let parsingResult = JSONC.parse(jsoncString, errors, {allowTrailingComma: true});
+    if(errors.length > 0){
+        let {error: errorCode, offset} = errors[0];
+        let errName = JSONC.printParseErrorCode(errorCode);
+        console.error({errName, offset});
+        throw new Error(`Cannot parse ${fileName ?? "JSONC"}: got ${errName} at position ${offset}`);
```

Comment:
Hmm, I think filename is already printed at a higher level (lines 34-35 in this file in `getTsConfig` function)

Omitting the second parameter would lead to a cleaner interface for this function and remove duplicate filenames in error message.

```
parcel-resolver-ts-base-url: Unexpected exception when reading jsonc-in-tsconfig/tsconfig.json: Cannot parse 
/Users/yakovlev-alex/Repos/parcel-resolver-ts-paths/tests/jsonc-in-tsconfig/tsconfig.json: got UnexpectedEndOfString at position 120
💡 Check if a tsconfig.json file exists and has valid configuration in it
```

The only difference I see now is here it prints absolute path while in `getTsConfig` it prints path relative to the project root. Now that I think about it, IDEs work better with absolute paths and can get confused by relative paths in workspaces with duplicated directory names like monorepos.

So can you please remove the filename here and make the filename in `getTsConfig` catch block absolute?